src/olympia/devhub/views.py

```diff
@@ -699,15 +699,14 @@ def handle_upload(
     submit=False,
     source=amo.UPLOAD_SOURCE_DEVHUB,
 ):
-    user = request.user if request.user.is_authenticated else None
```

Comment:
I assumed this was for the standalone upload case (just submitting to test validation) - I forget, do we support that unauthenticated these days?

Reply:
No, it requires auth.